src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -1363,6 +1389,21 @@ private void visitTypeValueAlias(String unqualifiedName, ObjectType otype) {
       typesUsed.add(otype.getDisplayName());
     }
 
+    private void visitTypeValueAlias(String unqualifiedName, String emitName) {
```

Comment:
try reusing visitKnownTypeValueAlias

Reply:
Yep, that works - thanks for pointing it out!